source/asynch_socket.c

```diff
@@ -30,6 +30,21 @@
 
 #include "sal-stack-lwip/lwipv4_init.h"
 
+/**
+ * LwIP is not re-entrant for any APIs except pbuf and memp APIs.
+ * To correct for this, all packet processing should be deferred to event context.
+ * However, since sal-driver-lwip-k64f does all packet processing in IRQ context,
+ * we must use the same paradigm here until sal-driver-lwip-k64f is updated to defer
+ * packet processing to event context.
+ */
+#ifdef YOTTA_SAL_DRIVER_LWIP_K64F_ETH_VERSION_STRING
+#include "uvisor-lib/uvisor_exports.h"
+#include "uvisor-lib/interrupts.h"
+
+extern volatile uint8_t emac_timer_fired;
+extern struct pbuf * volatile emac_tcp_push_pcb;
```

Comment:
Make it look like you added the `extern` properly from the first time. (Squash the "extern" commit with " Add a workaround for outputting tcp buffers …")